_docs/_latest/configure/explain/index.md

```diff
@@ -1,135 +1,59 @@
 ---
 title: Explain
-order: 202
+order: 000
 ---
 # {{ page.title }}
 
-Explain is not part of the main Forseti releases from master. Please
-deploy from the Explain experimental branch.
+This guide describes how to set up Explain for Forseti Security.
```

Comment:
New line needed after the first sentence

Reply:
Updated.